modules/runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/tools/command/PositionCommand.java

```diff
@@ -212,7 +224,7 @@ protected boolean positionAfterDate(LogManager manager, Name group, Name name, i
                 }
                 tailer.seek(logOffset);
                 movedOffset = true;
-                log.info(String.format("# Set log %s, group: %s, to offset %s", labelFor(name, part), group,
+                log.warn(String.format("# Set log %s, group: %s, to offset %s", labelFor(name, part), group,
```

Comment:
Why? Is it an abnormal behavior?

Reply:
it is not abnormal, but moving consumer offset is an important administrative operation so it is important to warn